docs/sdkx-cocos2dx/users.mdx

```diff
@@ -0,0 +1,301 @@
+        <li> If the value of "userId" is empty string, after trimming spaces, then do not include this parameter in "login" call. </li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>userEmail</td>
+    <td>Required as an identifier if the user ID is not provided</td>
+    <td>A user's email address. If you don't know the user's email, you can leave it blank and the Identity Bot (when enabled) will ask the user for their email. If you provide a value, then the Identity Bot will not ask the user for their email again.</td>
+    <td>
+      <ul>
+        <li>Email format should be valid (should be in the format "my@email.com"). Invalid emails will result in the creation of an anonymous user.</li>
+        <li>Leading / trailing spaces are not allowed. Emails with leading / trailing spaces will result in the creation of an anonymous user.</li>
+        <li>Email is always case insensitive (eg. 'MY@EMAIL.COM' is same as 'my@email.com').</li>
+        <li> If the value of "userEmail" is empty string, after trimming spaces, then do not include this parameter in "login" call. </li>
+      </ul>
+</td>
+  </tr>
+  <tr>
+    <td>userAuthToken</td>
+    <td>Optional</td>
+    <td>A user authentication token generated via Hash-Based Message Authentication Code (HMAC) using SHA256. You should provide an HMAC Digest if you want to ensure that 3rd parties cannot file issues on behalf of your users or update their properties. Details <a href="https://support.helpshift.com/kb/article/what-is-user-identity-verification-and-how-do-i-set-it-up/">here</a>.</td>
+    <td>
+      <ul>
+        <li>A valid HMAC Digest should be provided in order to ensure your users can file issues. Details <a href="https://support.helpshift.com/kb/article/how-do-i-test-that-i-have-user-identity-verification-set-up-correctly/">here</a>.</li>
+        <li> If the value of "userAuthToken" is empty string, after trimming spaces, then do not include this parameter in "login" call. </li>
+      </ul>
+</td>
+
+  </tr>
+</table>
+
+You need to be aware of some considerations when using the `HelpshiftCocos2dx::login()` API -
+
+- In order to create a logged-in user in Helpshift, the use of either `userId` or `userEmail` is required.
+- The `userId` or `userEmail` should uniquely identify the user across all users of your app. It should **NOT** be duplicated for two or more different users.
+- If the login API is called with different user identifier, then it first logs out the currently logged-in user and then logs in with this user identifier.
+- It is best to call login immediately when your app user logs in.
+- If you are setting `fullPrivacy` to true, you shouldn't use email as the only identifier in the login API for that user. This will result in the creation of an anonymous user.
+
+Example:
+
+```cpp
```

Comment:
Check the corresponding code snippet in native iOS/Android/Xamarin and follow the same structure here. This change was done after work on cocos docs started.